pkg/recycle/drain_test.go

```diff
@@ -146,6 +150,111 @@ func TestRecycler_drainNode(t *testing.T) {
 	assert.Nil(t, podsOnNode.Items)
 }
 
+func TestRecycler_drainNodeWithPDB(t *testing.T) {
+
```

Comment:
It may seem way over opinionated, but can we remove this empty line please?

Reply:
Updated Jason